eddymotion/model.py

```diff
@@ -91,6 +91,24 @@ def predict(self, gradient, **kwargs):
         return self._S0
 
 
+class AverageDWModel:
+    """A trivial model that returns an average map."""
+
+    __slots__ = ("_data",)
+
+    def __init__(self, gtab, **kwargs):
+        """Implement object initialization."""
+        return  # do nothing at initialization time
```

Comment:
```suggestion
        self._gtab = gtab
```

Reply:
keep a reference to the gradient table, so that later you can check on it when creating the average.